src/Events/Calendar_Embeds/Controller.php

```diff
@@ -116,6 +139,44 @@ public function register_menu_item() {
 	 * @return string
 	 */
 	public function keep_parent_menu_open( $submenu_file ) {
-		return $this->container->make( Calendar_Embeds::class )->keep_parent_menu_open( $submenu_file );
+		return $this->container->make( Admin\Page::class )->keep_parent_menu_open( $submenu_file );
+	}
+
+	/**
+	 * Manage column ids and headers for the admin list table.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $columns The columns.
+	 *
+	 * @return array
+	 */
+	public function manage_table_columns( $columns ): array {
+		return $this->container->make( Admin\List_Table::class )->manage_columns( $columns );
+	}
+
+	/**
+	 * Manage column content for the admin list table.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $column  The column name.
+	 * @param int    $post_id The post ID.
+	 *
+	 * @return void
+	 */
+	public function manage_table_column_content( $column, $post_id ): void {
+		$this->container->make( Admin\List_Table::class )->manage_column_content( $column, $post_id );
+	}
+
+	/**
+	 * Enqueue assets for the admin page.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function enqueue_admin_page_assets(): void {
+		$this->container->make( Admin\Page::class )->register_assets();
```

Comment:
Why are we still going with the fake actions that redirect to the real actions ? Having an abstraction layer that only offers more code to maintain... Not optimal

Reply:
All the hooks are in one place is against my preference. If that is TEC policy though ill oblige :D

I am more of the idea "the hooks are in their appropriate place"

This code is just redirecting to where the hooks should be to begin with.

Reply:
Of course that isn't strict and code should also include some level of creativity IMO. What i would always vote against is not needed code!

Anyway, not going to force anything to change or continue to be mentioning in every PR.